### PR TITLE
Introduce interface for clearing visitor session

### PR DESCRIPTION
**Pull request by dukhovnyi**
Added "Clear Session" button on UI that removes visitor and cleans up chat history.
MUIC-587

### CODE REVIEW

File: app/src/main/res/layout/main_fragment.xml [hunk 1]
```diff
@@ -88,6 +88,16 @@
         app:layout_constraintStart_toStartOf="@id/start_guideline"
         app:layout_constraintTop_toBottomOf="@id/audio_call_button" />
 
+    <Button
+        android:id="@+id/clear_session_button"
+        android:layout_width="0dp"
+        android:layout_height="51dp"
+        android:layout_marginTop="8dp"
```
Comment (deniss-mc) [inline app/src/main/res/layout/main_fragment.xml:95]: use `@dimen/margin_m` here

File: app/src/main/res/layout/main_fragment.xml [hunk 2]
```diff
@@ -88,6 +88,16 @@
...
+        android:text="Clear Session"
```
Comment (deniss-mc) [inline app/src/main/res/layout/main_fragment.xml:96]: Extract this to `strings.xml`

**Review by deniss-mc — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/main_fragment.xml (hunk 2 above)
Comment (dukhovnyi) [reply app/src/main/res/layout/main_fragment.xml]: fixed - b28f19808b0641620ed986b8fa412dfcd2597739

File: app/src/main/res/layout/main_fragment.xml (hunk 1 above)
Comment (dukhovnyi) [reply app/src/main/res/layout/main_fragment.xml]: fixed - b28f19808b0641620ed986b8fa412dfcd2597739

File: app/src/main/res/layout/main_fragment.xml [hunk 3]
```diff
@@ -88,6 +88,16 @@
         app:layout_constraintStart_toStartOf="@id/start_guideline"
         app:layout_constraintTop_toBottomOf="@id/audio_call_button" />
 
+    <Button
+        android:id="@+id/clear_session_button"
+        android:layout_width="0dp"
+        android:layout_height="51dp"
```
[the remark this answers is not in the document]
Comment (dukhovnyi) [reply app/src/main/res/layout/main_fragment.xml]: fixed - b28f19808b0641620ed986b8fa412dfcd2597739

**Review by deniss-mc — APPROVED** [review]
LGTM

**Review by AndriiHorishniiMOC — COMMENTED** [review]
It looks good for me

**Pull request merged**